### PR TITLE
(Partner ID: cyware-238) - CTIX new V3 app for XSOAR

### PR DESCRIPTION
**Pull request by gladipero**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by cywaredev** [comment]
@edik24 @ShahafBenYakir @JasBeilin Have signed the CLA using both mine and the cywaredev account, Not sure what I am missing here, any help will be appreciated.

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Thank you for contributing! I have provided you with some feedback below:

1. Deleting pipfile and pipfile.lock is advised, as it is not being used.
2. In order to avoid confusing users, the command names should begin with integration names and include ‘-’ instead of ‘_’.
If you're using `get_indicator_details`, you should use `ctix-get-indicator-details`.
3. In cases where no results are returned, it is advisable to handle them gracefully. *get_lookup_threat_data_command*, for example, should return 'No results were found' as readable_output. If not, customers might think the data was not loaded correctly.

File: Packs/Tests/id_set.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (JasBeilin) [inline Packs/Tests/id_set.json:1]: Please remove this file from your changes.

File: Packs/CTIX/Tests/id_set.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (JasBeilin) [inline Packs/CTIX/Tests/id_set.json:1]: Please remove this file from your changes.

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Please note that validation and unitests are failing. 
If you need help with addressing these issue, feel free to reach us.

**Comment by cywaredev** [comment]
@JasBeilin @Noy-Maimon @edik24 All changes requested are done, could you please verify. Also not sure how to get that CLA step done as I have already signed using both the cywaredev as well as gladipero github accounts.

**Comment by Noy-Maimon** [comment]
@cywaredev Great work, please note that the first two comment here still needs to be changed https://github.com/demisto/content/pull/19060#pullrequestreview-984668566

**Comment by cywaredev** [comment]
Hey @Noy-Maimon Thanks for pointing it out, the change is done now.

**Comment by cywaredev** [comment]
@Noy-Maimon It's done, let me know if any more changes need to be done. Thanks!

**Comment by Noy-Maimon** [comment]
@cywaredev Are you sure you pulled the changes from our master to? as in, the repo your main repo was forked from. I checked out your branch and it seems it is still not updated.

**Comment by cywaredev** [comment]
Screenshot 2022-05-26 at 6 53 26 PM
@Noy-Maimon Can you check? Github shows no commits to be pulled from demisto master.

**Comment by Noy-Maimon** [comment]
I checked out your branch and i can see that it is not updated - the file .gitlab/ci/bucket-upload.yml was modified today and those changes are not there (and those changes are a part of what's blocking the build). I can push those changes for you but i need editing permissions for your branch. If you could please supply that for my user, I will solve that for you.

**Comment by Noy-Maimon** [comment]
> 

@cywaredev By the way, regarding the CLA, the user Amar Shishodia should sign the [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=19060) form.

**Comment by cywaredev** [comment]
Screenshot 2022-05-27 at 1 57 02 PM
@Noy-Maimon I have accepted using the Amar Shishodia github but CLA is not detecting the same.
Also the file of gitlab is now synced to be the one last updated 21 hours ago and should unblock your pipelines now.

**Comment by cywaredev** [comment]
@Noy-Maimon @JasBeilin I found the issue with the CLA to solve this created a new PR with exact same code here. Request you to proceed in that PR.
https://github.com/demisto/content/pull/19233

**Pull request closed without merging**